concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1447,6 +1458,40 @@ jobs:
                   cf enable-feature-flag diego_docker
 
       - aggregate:
+        - task: deploy-compose-service-broker
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+                tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
+            inputs:
+              - name: paas-cf
+              - name: paas-compose-broker
+              - name: config
+              - name: bosh-CA
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                  . ./config/config.sh
+                  echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                  cf target -o service-brokers -s compose
+
+                  cp ./paas-cf/config/service-brokers/compose/catalog.json ./paas-compose-broker/
+                  cd paas-compose-broker/
+                  cf push --no-start
+                  cf set-env compose-broker LOG_LEVEL DEBUG
+                  cf set-env compose-broker USERNAME compose-broker
+                  cf set-env compose-broker PASSWORD "$COMPOSE_BROKER_PASS"
+                  cf blue-green-deploy compose-broker
```

Comment:
This is done. We have short ruby script to add required env variables to the manifest.